client/ayon_core/hosts/max/plugins/load/load_max_scene.py

```diff
@@ -14,6 +16,56 @@
 from ayon_core.pipeline import get_representation_path, load
 
 
+class MaterialDupOptionsWindow(QtWidgets.QDialog):
+
+    def __init__(self, material_options):
+        super(MaterialDupOptionsWindow, self).__init__()
+        self.setWindowFlags(self.windowFlags() | QtCore.Qt.FramelessWindowHint)
+
+        self.material_option = None
+        self.material_options = material_options
+
+        self.widgets = {
+            "label": QtWidgets.QLabel(
+                "Select material duplicate options before loading the max scene."),
+            "material_options_list": QtWidgets.QListWidget(),
+            "warning": QtWidgets.QLabel("No material options selected!"),
+            "buttons": QtWidgets.QWidget(),
+            "okButton": QtWidgets.QPushButton("Ok"),
+            "cancelButton": QtWidgets.QPushButton("Cancel")
+        }
+        for option in material_options.values():
+            self.widgets["material_options_list"].addItem(option)
+        # Build buttons.
+        layout = QtWidgets.QHBoxLayout(self.widgets["buttons"])
+        layout.addWidget(self.widgets["okButton"])
+        layout.addWidget(self.widgets["cancelButton"])
+        # Build layout.
+        layout = QtWidgets.QVBoxLayout(self)
+        layout.addWidget(self.widgets["label"])
+        layout.addWidget(self.widgets["material_options_list"])
+        layout.addWidget(self.widgets["buttons"])
+
+        self.widgets["okButton"].pressed.connect(self.on_ok_pressed)
+        self.widgets["cancelButton"].pressed.connect(self.on_cancel_pressed)
+        self.widgets["material_options_list"].itemPressed.connect(
+            self.on_material_optionsPressed)
+
+    def on_material_optionsPressed(self, item):
+        self.material_option = next((key for key in self.material_options.keys()
+                                     if self.material_options[key]== item.text()),
+                                     None)
```

Comment:
This would be so much nicer if we could match the item by key instead of the value. Why does the option match the value instead of the key which is what we should be matching against instead?
```suggestion
        self.material_option = self.material_options.get(item.text(), None)
```

If we would append QListWidgetItem instead  of just the text

```python
item = QtWidgets.QListWidgetItem(text)
item.setData(QtCore.Qt.UserRole, key)
```

Then here we could just do:
```suggestion
        self.material_option = item.data(QtCore.Qt.UserRole)
```